index.html

```diff
@@ -352,21 +351,22 @@ <h3>Ecosystem Overview</h3>
           </dd>
           <dt><a>verifier</a></dt>
           <dd>
-A role an <a>entity</a> might perform by requesting and receiving a
-<a>verifiable presentation</a> that proves the <a>holder</a> possesses the
-required <a>verifiable credentials</a> with certain characteristics. Example
-verifiers include employers, security personnel, and websites.
+A role an <a>entity</a> might perform by receiving one or more
+<a>verifiable credentials</a>, optionally inside a
+<a>verifiable presentations</a>, for processing. Example verifiers include
```

Comment:
"inside a verifiable presentations" could better be "inside a verifiable presentation" or "inside verifiable presentations"

Reply:
fixed in [ede9787](https://github.com/w3c/vc-data-model/pull/508/commits/ede978791a48a2f05d8998c7f19ef204e20aed98)